techmatters/flex-plugins
# Add a view of referrals for contacts

This PR is related to https://github.com/techmatters/hrm/pull/301.

## Description
This PR adds support for consuming `referrals` associated to contacts in the contact details view. The different places where this should work as intended are:
- Contact details, from 
  - standalone contact search.
  - contact search within an in-progress task. 
- Case -> connected contact details, from 
  - standalone case search.
  - case search within an in-progress task.
  - case list.
- Case -> connected contact, for an in-progress task. 

### Checklist
- [x] [Corresponding issue has been opened](https://tech-matters.atlassian.net/browse/CHI-1422)
- [n/a] Feature flags added :bangbang: Do we want to hide this under a FF?

### Verification steps
- Start the test DB and local server and expose via ngrok (using the above linked version).
- Start a Flex local server.
- Save a new contact with some referrals with it.
- Confirm that the referrals are properly retrieved with the contact in the different views above mentioned.
